include/llvm/ADT/Triple.h

```diff
@@ -682,7 +682,7 @@ class Triple {
 
   /// Tests whether the target uses emulated TLS as default.
   bool hasDefaultEmulatedTLS() const {
-    return isAndroid() || isOSOpenBSD() || isWindowsCygwinEnvironment();
+    return /* LDC isAndroid() ||*/ isOSOpenBSD() || isWindowsCygwinEnvironment();
```

Comment:
Can't we fix this on the LDC side? I noticed that there are `llvm::TargetOptions::{EmulatedTLS,ExplicitEmulatedTLS}`.

Reply:
Yeah, just being lazy in figuring out that mess, :) this is the easiest fix that works. Since we're the only ones using this LLVM, rather just do it this way.

Reply:
No, you're right. I was stupidly thinking that I'd have to fix it elsewhere in LLVM, and misread what you wrote as that. Best to fix it in LDC as you said, will do that.